CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
-## [v4.10.0] - 2023-10-30
```

Comment:
also change log PR would always have only changes related to release, not other bump's ? like github version bump's. if we are following different strategy then fine. Just calling it out.

Reply:
sure i'll add the other PRs here